### ISSUE TITLE
add egoa split?

### ISSUE THREAD

**Issue by pete-hulson** [afsc-assessments/afscdata#43]
https://github.com/afsc-assessments/afscdata/blob/c49e463182d64bfa436692073db64010b6898e09/R/goa_rockfish.R#L110
any chance we could add a query for the egoa split fraction here?

**Comment by BenWilliams-NOAA** [comment]
can you put in an example of the old query?

**Comment by pete-hulson** [comment]
no... i would always just pull it from akfin in the 'end of assessment flurry'...

**Comment by BenWilliams-NOAA** [comment]
heh, for my clarification - this would be pulling the survey abundance out by area for a rema analysis or something different?

**Comment by BenWilliams-NOAA** [comment]
could you do this using `q_bts_biomass (type='area')`?

**Comment by pete-hulson** [comment]
no - this is a separate table from any of the standard biomass tables - which makes me think that we'd need to do this for dusky's too
